data/korath outfits.txt

```diff
@@ -318,4 +318,4 @@ outfit "Korath Repeater Rifle"
 	"capture attack" 1.6
 	"capture defense" 2.4
 	"unplunderable" 1
-	description "This rifle is commonplace among Korath ship crews and predates the creation of the Repeater used on Korath ships by at least two centuries. Since it uses superheated plasma instead of a laser it is slow to fire and will sometimes burn the users hands if not careful"
+	description "This rifle is commonplace among the Korath and predates the creation of the Repeater by at least two centuries. By using superheated plasma instead of lasers, The rifle is slow to fire and can sometimes burn the users hands if they aren't careful while making up for it by almost guaranteeing those hit won't live long enough to fire back."
```

Comment:
as said on Discord:
"The rifle" while there was a comma before "The"
and the last sentence is way too long

Reply:
thanks again